service/items.go

```diff
@@ -6,54 +6,51 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/rodrigomkd/go-rest-api/common"
-	"github.com/rodrigomkd/go-rest-api/config"
+	"github.com/rodrigomkd/go-rest-api/model"
+	"github.com/rodrigomkd/go-rest-api/service/api"
+	"github.com/rodrigomkd/go-rest-api/service/csv"
 )
 
-type item struct {
-	ID          int    `json:"ID"`
-	Description string `json:"Description"`
+//struct
+type Service struct {
+	cs         csv.CSVService
```

Comment:
Don't use the actual structures, instead create interfaces here that contain the methods that they're using. Don't use the interfaces you created on the other packages, interfaces should be defined were you use them, not were they are implemented